pkg/universe.dagger.io/alpha/kubernetes/helm/upgrade.cue

```diff
@@ -9,9 +9,15 @@ import (
 )
 
 #Upgrade: {
+	_base: #Image
+
 	// The image to use when running the action.
 	// Must contain the helm binary. Defaults to alpine/helm
-	image: *#Image | docker.#Image
+	image: *_base.output | docker.#Image
```

Comment:
I also tried to fix the image here. So that both work the default image and a user supplied image. I am not sure if this leads to the default image always being pulled, though.